.github/workflows/dbdocs.yml

```diff
@@ -0,0 +1,42 @@
+# This workflow generates database documentation and ERD files.
+#
+name: Generate dbdocs
+
+on:
+  pull_request:
+    paths:
+      - .github/workflows/dbdocs.yml
+      - db/migrations/bcda
+      - db/migrations/bcda_queue
+
+env:
+  VAULT_PW: ${{ secrets.VAULT_PW }}
+
+jobs:
+  generate-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          ref: ${{ github.event.pull_request.head.ref }}
+
+      - name: Decrypt secrets
+        run: |
+          echo $VAULT_PW > .vault_password
+          bash ops/secrets --decrypt
+          mv -fv shared_files/encrypted/* shared_files/decrypted/
```

Comment:
These github actions dont leave any kind of artifacts anywhere I assume?

Reply:
oh hm! I don't think so, I copied this from the existing ci-workflow.yml. We're good to keep decrypted secrets on the public GA machines, right @CMSgov/ab2d-bcda-dpc-platform?

Reply:
Better not to leave them lying around if we can avoid it. There was a [vulnerability found recently in GitHub Actions artifacts](https://unit42.paloaltonetworks.com/github-repo-artifacts-leak-tokens/). That doesn't apply directly here, but does point toward the potential for leaks.